src/components/collaborator_list/CollaboratorAddEdit.tsx

```diff
@@ -100,7 +114,11 @@ export default function CollaboratorAddEdit(props: CollaboratorAddEditProps): Re
                         type='button'
                         onClick={() => {
                             if (id < 0) {
-                                onCollaboratorChange([...collaborators, newCollaborator]);
+                                if (newCollaborator) {
+                                    onCollaboratorChange([...collaborators, newCollaborator]);
+                                } else {
+                                    onCollaboratorChange([...collaborators])
```

Comment:
Can you explain how this condition is possible? When `id < 0`, this means that we're creating a new collaborator, so `newCollaborator` should always be defined.